app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\User\CreateFormRequest;
+use App\Http\Service\UserService;
+use Illuminate\Support\Facades\Session;
+
+class UserController extends Controller
+{
+    protected $userService;
+
+    public function __construct(UserService $userService)
+    {
+        $this->userService = $userService;
+    }
+
+    public function index()
+    {
+//        session()->forget('users');
+        return view('admin.user.index',['users' => Session::get('users')]);
+    }
+
+    public function create()
+    {
+        return view('admin.user.add');
+    }
+
+    public function store(CreateFormRequest $request)
+    {
+
+        $this->userService->create($request->only('name','email'));
```

Comment:
`$this->userService->create($request->only('name','email'));`
=>

`$this->userService->create($request->only('name', 'email'));`